src/block_leaky_relu.rs

```diff
@@ -0,0 +1,180 @@
+use std::any::Any;
+use std::error::Error;
+
+use crate::block_helpers;
+use crate::feature_buffer;
+use crate::graph;
+use crate::graph::BlockGraph;
+use crate::model_instance;
+use crate::port_buffer;
+use crate::regressor;
+use regressor::BlockTrait;
+
+pub struct BlockLeakyRELU {
+    pub num_inputs: usize,
+    pub input_offset: usize,
+    pub output_offset: usize,
+    pub alpha: f32,
+}
+
+pub fn new_leaky_relu_block(
+    bg: &mut BlockGraph,
+    input: graph::BlockPtrOutput,
+) -> Result<graph::BlockPtrOutput, Box<dyn Error>> {
+    let num_inputs = bg.get_num_output_values(vec![&input]);
+    assert!(num_inputs != 0);
```

Comment:
frankly I just copy-pasted this from block_relu, that's why I said we might want to consider eliminating code repetition.
as to whether this should be a assert of debug_assert - since we run FW from release builds, and it seems @andraztori intended for this to fail in case of bad wiring, I think we should keep it a regular assert for now.